rubocop/jekyll/no_p_allowed.rb

```diff
@@ -0,0 +1,23 @@
+# frozen_string_literal: true
+
+require "rubocop"
+
+module RuboCop
+  module Cop
+    module Jekyll
+      class NoPAllowed < Cop
+        MSG = "Avoid using p to print things".freeze
```

Comment:
Can you use the same message as in `no_puts_allowed` that suggests Jekyll.logger instead?